manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -39,8 +39,6 @@ spec:
               - ALL
             privileged: false
             readOnlyRootFilesystem: true
-            runAsGroup: 1337
-            runAsUser: 1337
```

Comment:
Why is this removed?

Reply:
(same question as above)

Reply:
I think we can resolve this conversation?